Resources/doc/3-listener-support.rst

```diff
@@ -184,7 +196,7 @@ Listener                    Event                  Priority
 ==========================  =====================  ========
 ``MimeTypeListener``        ``kernel.request``     200
 ``FormatListener``          ``kernel.request``     34
-``VersionListener``         ``kernel.request``     33
+``VersionListener``         ``kernel.request``     31
```

Comment:
Hum... Well, we can maybe remove the uri resolver, for most of the cases it only duplicates the corresponding route attribute.